app/views/offers/new.html.erb

```diff
@@ -12,9 +12,8 @@
     <%= f.input :start_date %>
     <%= f.input :end_date %>
     <%= f.input :photos, as: :file, input_html: { multiple: true } %>
-    <%= f.button :submit,"Submit" class:"btn-smpl-form" %>
+    <%= f.button :submit, class:"btn-smpl-form" %>
+    <%= link_to "cancel", overview_path, :class=>"btn-ghost" %>
```

Comment:
:class => ... is old syntax; best to write in the new way / class: '...'

Reply:
done!